packages/client/src/features/tools/change-bounds/change-bounds-tool.ts

```diff
@@ -183,7 +167,7 @@ export class ChangeBoundsListener extends DragAwareMouseListener implements ISel
                 const resizeActions = this.handleResizeOnClient(positionUpdate);
                 actions.push(...resizeActions);
             }
-            this.tool.dispatchFeedback(actions, this);
+            return actions;
```

Comment:
This does revert the change of https://github.com/eclipse-glsp/glsp-client/pull/262.
The idea was to dispatch the bounds changes as feedback so that they don't get reset if you receive a server update while resizing.